tests/index.test.js

```diff
@@ -249,6 +249,30 @@ describe('index.js', () => {
       assert.deepEqual(JSON.parse(output.body), expectedResult, 'Output data does not match expected.');
       assert.strictEqual(output.statusCode, 200, 'Status code should be 200');
     });
+    it('check call to ANY handler with HttpApiV2 event object', async () => {
+      let pathResolverMock = sandbox.mock(PathResolver.prototype);
+      pathResolverMock.expects('storePath').returns({});
+      pathResolverMock.expects('resolvePath').returns(null);
+      let expectedResult = { value: 5 };
+      let api = new Api(null, noopFunction);
+      api.any('/test', () => {
+        return new Response(expectedResult);
+      });
+      const event = {
```

Comment:
I would expect this event to match what AWS suggests here: https://docs.aws.amazon.com/apigateway/latest/developerguide/http-api-develop-integrations-lambda.html

Is their documentation wrong?

Reply:
this https://github.com/awsdocs/aws-lambda-developer-guide/blob/main/sample-apps/nodejs-apig/event-v2.json seems like a better example